src/main/java/Helper.java

```diff
@@ -0,0 +1,122 @@
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.FormatStyle;
+import java.util.ArrayList;
+
+public class Helper extends Duke{
+    public static void addToDo(String[] line, ArrayList<Task> taskList) throws DukeException {
+        String description = line[1];
+        // Validate description
+        if (description.isEmpty()) {
+            throw new DukeException("☹ OOPS!!! Description of todo unclear. Here's an example: 'todo borrow book'");
+        }
+        Task task = new ToDo(description);
+        taskList.add(task);
+        ReadWriteFile.writeToFile(taskList);
+
+        System.out.println("Got it. I've added this task:");
+        System.out.println(task);
+        System.out.println("Now you have " + taskList.size() + " task(s) in the list.");
+    }
+
+    public static void showList(ArrayList<Task> taskList) throws DukeException {
+        if (taskList.isEmpty()) {
+            throw new DukeException("The list is empty! Add in some new tasks!");
+        }
+        int counter = 1;
+        for (Task item : taskList) {
+            System.out.println(counter + ". " + item);
+            counter++;
+        }
+    }
+
+    public static void markAsDone(String[] line, ArrayList<Task> taskList) throws DukeException{
```

Comment:
Should there be whitespace in-between`DukeException` and `{`?

Reply:
Yes too! Thanks!